scipy/special/add_newdocs.py

```diff
@@ -2263,10 +2263,11 @@ def add_newdoc(place, name, doc):
     """
     expit(x)
 
-    Expit ufunc for ndarrays.
+    Expit (a.k.a. logistic sigmoid) ufunc for ndarrays.
 
-    The expit function, also known as the logistic function, is defined as
-    expit(x) = 1/(1+exp(-x)). It is the inverse of the logit function.
+    The expit function, also known as the logistic sigmoid function, is
+    defined as expit(x) = 1/(1+exp(-x)).  It is the inverse of the logit
```

Comment:
Use math mode or double backticks for the expit expression?

Reply:
I left this as it was.  We used to have a policy of restricting `:math:` to the Notes section of the docstring (and I agree with that policy), but that doesn't seem to be followed closely these days.  The expression isn't really code, so double backticks don't improve the readability of the text.  But you're not the only one who will probably suggest doing *something* with that expression, so I'll stick the backticks in there.

Reply:
I added backticks around the expression.